auslib/test/blobs/test_guardian.py

```diff
@@ -37,7 +40,14 @@ def testContainsForbiddenDomain(guardianblob, whitelistedDomains, expected):
     "version,expected", [("0.5.0.0", True), ("0.8.0.0", True), ("0.99.99.99", True), ("1.0.0.0", False), ("1.0.5.0", False), ("2.0.0.0", False)]
 )
 def testShouldServeUpdateVariousVersions(guardianblob, version, expected):
-    updateQuery = {"product": "Guardian", "version": version, "buildTarget": "WINNT_x86_64", "channel": "release"}
+    updateQuery = {
+        "product": "Guardian",
+        "version": version,
+        "buildTarget": "WINNT_x86_64",
+        "channel": "release",
+        "hashFunction": "hash512",
+        "hashValue": "abcdef",
```

Comment:
Not sure why these needed to be added, they're not part of the update query. Please remove.

Reply:
Oops!